content/manuals/subscription/core-subscription/downgrade.md

```diff
@@ -34,21 +34,21 @@ If you want to downgrade a Docker Business subscription and your organization us
 
 >[!IMPORTANT]
 >
->If you have a [sales-assisted Docker Business subscription](details.md#sales-assisted), contact your account manager to downgrade your subscription. 
+>If you have a [sales-assisted Docker Business subscription](details.md#sales-assisted), contact your account manager to downgrade your subscription.
 
-1. Sign in to your [Docker Hub](https://hub.docker.com) account.
+1. Sign in to [Docker Home](https://app.docker.com).
 
-2. Select your avatar in the top-right corner and from the drop-down menu select **Billing**.
+2. Select your **avatar** and from the drop-down menu select **Billing**.
 
-3. Choose either your personal account or an organization to downgrade. 
+3. Choose either your personal account or an organization to downgrade.
 
 4. In the **Plan** section, select **Change plan**.
 
-5. On the **Change Plan** page, select the plan you'd like to downgrade to. 
+5. On the **Change Plan** page, select the plan you'd like to downgrade to.
```

Comment:
Not part of the requested change, but it looks like this UI text got changed at some point. The page title appears to be dynamic based on the plan selected. Maybe don't mention the page name.

```suggestion
5. Select the plan you'd like to downgrade to.
```

Or, combine step 4 and 5, like the upgrade flow.

4. On the **Billing Details** tab, select **Change plan** and then choose the plan you'd like to downgrade to.